src/node.js

```diff
@@ -388,17 +389,17 @@
       tickDone();
     }
 
+    function PObj(c) {
+      this.callback = c;
+      this.domain = process.domain || null;
+    }
+
     function nextTick(callback) {
       // on the way out, don't bother. it won't get fired anyway.
       if (process._exiting)
         return;
 
-      var obj = {
-        callback: callback,
-        domain: process.domain || null
-      };
```

Comment:
As I understand it, V8 does a better job creating hidden classes for objects constructed with a function constructor / can pre-allocate the expected number of properties for them.